pkg/apiserver/authenticator/authn.go

```diff
@@ -96,12 +99,20 @@ func New(config AuthenticatorConfig) (authenticator.Request, error) {
 		authenticators = append(authenticators, serviceAccountAuth)
 	}
 
-	if len(config.KeystoneURL) > 0 {
-		keystoneAuth, err := newAuthenticatorFromKeystoneURL(config.KeystoneURL)
-		if err != nil {
-			return nil, err
+	if len(config.KeystoneConfig) > 0 {
```

Comment:
I tried that originally, but it would try and authenticate to both, not just one, which always broke. If there is a way to enable them both at the same time, that would be great. Maybe I missed something?